File: templates/matlab/materials_microstructure_response_surface.m
function fig = materials_microstructure_response_surface()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('contour', 1804, 'materials microstructure: response contour surface', 'materials microstructure', 'response contour surface');
end

File: templates/python/materials_microstructure_response_surface.py
"""materials_microstructure_response_surface: 材料微结构响应等值面（contour 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='materials microstructure: response contour surface'):
    return make_template_figure('contour', seed=1804, title=title, domain='materials microstructure', topic='response contour surface')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
